Lib/ufo2ft/otfPostProcessor.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function, division, absolute_import, unicode_literals
+
+import tempfile
+
+from compreffor import Compreffor
+from fontTools.ttLib import TTFont
+
+
+class OTFPostProcessor(object):
+    """Does some post-processing operations on a compiled OpenType font, using
+    info from the source UFO where necessary.
+    """
+
+    def __init__(self, otf, ufo):
+        self.ufo = ufo
+        tmp_file = tempfile.NamedTemporaryFile()
+        tmp_path = tmp_file.name
+        otf.save(tmp_path)
```

Comment:
You should `seek(0)` before reloading the TTFont from the same BytesIO which you saved to.